packages/desktop-client/src/hooks/useServerURL.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+
+export function useServerURL() {
```

Comment:
question: this hook is now used in multiple components. Does the `get-server-url` request get dispatched each time the hook is used? If so - can we cache the results?

Reply:
It would be, yes. Given (see below) how it should be basically instantaneous, I don’t think the time spent on figuring out how to correctly invalidate the cache would be well-spent.